src/groupByKey.ts

```diff
@@ -2,6 +2,18 @@ type GroupsMap<T> = {
   [key: string]: T[];
 };
 
-export function groupByKey(items, key) {
-  // write code here;
+export function groupByKey<T>(items: T[], key: keyof T): GroupsMap<T> {
+  const result: GroupsMap<T> = {};
+
+  items.forEach((item) => {
+    const sortKey: string = String(item[key]);
```

Comment:
should work without explicit mention of the string type, TS will infer it from usage
```suggestion
    const sortKey = String(item[key]);
```